Update Mysterium Node's version (#428)
mysteriumnetwork/mysterium-vpn: README.md

## Build Setup

> **Note** that currently, the actually supported development environment is *OSX*. Other important ones like Plan9 and Linux is work in progress. And Windows usage in on your own risk.

Reviewer: BTW, have you actually tested this `myst` version bump on Windows machine?

Author: No, regression testing during release

Reviewer: :(

> **Note** that currently, the actually supported development environment is *OSX*. Other important ones like Plan9 and Linux is work in progress. And Windows usage in on your own risk.

Reviewer: > And Windows usage in on your own risk.

Majority of our users use Windows, so we test it - it's not "on your own risk" :D

Reviewer: ```suggestion
**Note** that currently, *macOS* has the best support - but we also support *Windows*.
Other important ones like Plan9 and Linux is also in our plans.
```

> **Note** that currently, the actually supported development environment is *OSX*. Other important ones like Plan9 and Linux is work in progress. And Windows usage in on your own risk.

Author: It's just a funny text about development environment :)

> **Note** that currently, the actually supported development environment is *OSX*. Other important ones like Plan9 and Linux is work in progress. And Windows usage in on your own risk.

Author: Text improved to be just text about development environment setup

* **Step 1.** Install project dependencies

Install `yarn` from https://yarnpkg.com/lang/en/docs/install/
Once you're all set, cd into your project's root directory.
```bash
brew install yarn

> yarn install

Author: I want to be more explicit here

> yarn install

Reviewer: `install` is redundant.

```suggestion
yarn
```

```

* **Step 2.** Install Mysterium Node dependency
```bash
yarn download:bins:osx
```

**Purpose of script:**

> - Grab the latest `myst` binary from https://github.com/mysteriumnetwork/node/releases and put it into your project's `bin` directory

Reviewer: This script doesn't download latest `myst` - it downloads the same version as `mysterium-client-bin`.
i.e. if you have `"mysterium-client-bin": "0.4.0"`, it will grap `0.4.0` version of myst from releases page.

- Install `openvpn` from https://openvpn.net/ or for OSX with homebrew run `brew install openvpn`

> - Download `update-resolv-conf` and `GeoLite2-Country.mmdb` from https://github.com/mysteriumnetwork/node/tree/master/bin/client_package/config and paste it into your project's `bin/config` directory.

Reviewer: Hmm, aren't `update-resolv-conf` and `GeoLite2-Country.mmdb` already embedded into myst?

Author: Updated docs
